### ISSUE TITLE
Split Items in Folders 

### ISSUE THREAD

**Issue by smarteez17** [ErikEJ/EFCorePowerTools#481]
Hi how can I split so that all the configuration files goes into the configuration  folder and DBContext file goes into the Context folder

**Comment by ErikEJ** [comment]
Duplicate of #423 ?

**Comment by smarteez17** [comment]
@ErikEJ 

![image](https://user-images.githubusercontent.com/35128168/91216082-abbf6c80-e715-11ea-85da-31d29cefde4c.png)


The context didnt add to the correct folder it added to config  but the namespace is correct 

![image](https://user-images.githubusercontent.com/35128168/91216211-ddd0ce80-e715-11ea-8fbd-1d92fe44713e.png)

**Comment by ErikEJ** [comment]
I am having trouble following, suggest you read the wiki post to understand the folder / namespace options.

**Comment by ErikEJ** [comment]
It would be useful if you posted/sharef an example of what you would like to achieve.

**Comment by smarteez17** [comment]
@ErikEJ 

I created a class Library and added 3 folders namely config , context and models . I would like all the models files to be places in the model folder , config files in the config folder and the context file in the context folder 

I used the tool as per the image above adding and ticking all the options I want. 

The output is that  all models files to be places in the model folder and all config files in the config folder. the context file is not placed into the context folder but its been placed in the config folder. When I open and look at the context file it has the correct namespace ie "ClassLibrary1.context" but it should be in the context folder
